### PR TITLE
[MAPSAND-605] update core styles

### PR DESCRIPTION
**Pull request by ank27**
### Summary of changes
Update core styles to latest version:
- MAPBOX_STREETS -> `mapbox://styles/mapbox/streets-v12`
- SATELLITE_STREETS -> `mapbox://styles/mapbox/satellite-streets-v12`
- OUTDOORS -> `mapbox://styles/mapbox/outdoors-v12`
- LIGHT -> `mapbox://styles/mapbox/light-v11`
- DARK -> `mapbox://styles/mapbox/dark-v11`

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Features ✨ and improvements 🏁
+* Update mapbox styles to latest versions. ([1837](https://github.com/mapbox/mapbox-maps-android/pull/1837))
```
Comment (kiryldz) [inline CHANGELOG.md:7]: afair we were planning to release that in v10.10? then it should move to rc section I assume

File: app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/MovingIconWithTrailingLineActivity.kt [hunk 2]
```diff
@@ -237,7 +237,7 @@ class MovingIconWithTrailingLineActivity : AppCompatActivity() {
         lineJoin(LineJoin.ROUND)
         lineWidth(4.0)
       },
-      below = "road-label"
+      below = "road-label-simple"
```
Comment (paulrenn67) [inline app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/MovingIconWithTrailingLineActivity.kt:240]: Move to constants

File: app/src/main/java/com/mapbox/maps/testapp/examples/VectorTileSourceActivity.kt [hunk 3]
```diff
@@ -33,7 +33,7 @@ class VectorTileSourceActivity : AppCompatActivity() {
             lineColor(Color.parseColor("#ff69b4"))
             lineWidth(1.9)
           },
-          below = "road-label"
+          below = "road-label-simple"
```
Comment (paulrenn67) [inline app/src/main/java/com/mapbox/maps/testapp/examples/VectorTileSourceActivity.kt:36]: Move to constants

**Comment by paulrenn67** [comment]
Does `road-label ` also need to be changed also in DistanceExpressionActivity.kt, VectorTileSourceActivity.kt, WithinExpressionActivity.kt?

**Comment by ank27** [comment]
Moving it to draft till we get final say on which version core styles will land.

**Comment by ank27** [comment]
closing it, not required now.

**Pull request closed without merging**